storage/src/mmr/verification.rs

```diff
@@ -26,7 +27,7 @@ pub trait Storage<H: CHasher> {
     fn size(&self) -> u64;
```

Comment:
Likewise, we may want to have this return an error?

Reply:
done